execution_workflows/DaPars/samplesheet_example_files.csv

```diff
@@ -0,0 +1,3 @@
+sample,condition,bam,bai
+control_r1,control,/Users/fzhuang/Desktop/APAeval/APAeval/execution_workflows/DaPars/../../tests/test_data/siControl_R1_2genes_Chr_prefix.bam,/Users/fzhuang/Desktop/APAeval/APAeval/execution_workflows/DaPars/../../tests/test_data/siControl_R1_2genes_Chr_prefix.bam.bai
+srsf3_r1,srsf3,/Users/fzhuang/Desktop/APAeval/APAeval/execution_workflows/DaPars/../../tests/test_data/siSrsf3_R1_2genes_Chr_prefix.bam,/Users/fzhuang/Desktop/APAeval/APAeval/execution_workflows/DaPars/../../tests/test_data/siSrsf3_R1_2genes_Chr_prefix.bam.bai
```

Comment:
Should replace your local paths with the path_to as mentioned in README and done by yourself for APAtrap :)